test/specs/loadPage.spec.ts

```diff
@@ -11,26 +11,23 @@ const inputType: string = categoryData.loadPageDataSet.inputType
 const input3: string = categoryData.loadPageDataSet.input3
 /* Data */
 
-describe("Loading Page", async () => 
-{
-    it("Should load page and perform assertions", async () => 
-    {
+describe("Loading Page", async () => {
+    it("Should load page and perform assertions", async () => {
         //Load Page And Perform Basic Assertions
         await category.open("")
-        await category.maxWin
+        await category.screenMaximize
         await category.getUrlAndTitle()
 
         //WebDriverIO Assertions
         await expect(browser).toHaveUrlContaining(input1)
         expect(await browser).toHaveTitleContaining(input2)
```

Comment:
instead of input1 and input2, can you name them properly?

Reply:
@gayatri-gawali  please resolve your comment if you are done with that , only then i can get to know that you have completed your review remarks